config/sample_application.yml

```diff
@@ -2,8 +2,6 @@ GITHUB_KEY:
 GITHUB_SECRET:
 GITHUB_TOKEN:
```

Comment:
@rhymes This also seems to be unused, it only occurs in 

* `Envfile`
* `docker-run.sh`
* `config/sample_application.yml`
* `docs/backend/auth-github.md`

Should I remove it too?

Reply:
Yeah, it's definitely unused. I think it ended up the in the docker run just because it was present in the Envfile

Reply:
Cool, will remove it tomorrow.